services/QuillLMS/engines/evidence/lib/evidence/evidence/synthetic/seed_data_generator.rb

```diff
@@ -11,16 +11,29 @@ class SeedDataGenerator
       PERIOD = '.'
       CSV_SUFFIX = '.csv'
 
-      FULL_COUNT = ENV.fetch('SYNTHETIC_SEED_PASSAGE_COUNT', 128).to_i
+      FULL_COUNT = ENV.fetch('SYNTHETIC_SEED_PASSAGE_COUNT', 50).to_i
       FULL_NOUN_COUNT = ENV.fetch('SYNTHETIC_SEED_NOUN_COUNT', 50).to_i
-      SECTION_COUNT = ENV.fetch('SYNTHETIC_SEED_SECTION_COUNT', 70).to_i
+      SECTION_COUNT = ENV.fetch('SYNTHETIC_SEED_SECTION_COUNT', 25).to_i
 
-      TEMPS_PASSAGE = [1, 0.9, 0.7, 0.5]
-      TEMP_SECTION = 0.5 # give a lower temp (creativity) when it has less info
+      TEMPS_PASSAGE = [0.8, 0.7, 0.5, 0.4]
```

Comment:
Note, there is a small lowering of temperature since we want slightly less creative answers than we are getting in the seed data.